docs/source/tutorial/schema.md

```diff
@@ -5,8 +5,314 @@ description: Start here for the Apollo fullstack tutorial
+
+```js
+type Rocket {
+  id: ID!
+  name: String!
+  type: String!
+}
+```
+
+This represents a rocket for a launch. An `ID`, `name`, and `type` fields are needed to identify a rocket.
+
+```js
+type User {
+  id: ID!
+  email: String!
+  trips: [Launch]!
+}
+```
+
+The `id` and `email` are fields for identifying a user. The `trips` field is necessary for providing easy access to the number of trips a user has booked.
+
+Let's now deal with the `Mutation` type.
+
+The `Mutation` type defines the entry points into Apollo server for modifying server data. For the `Mutation` type, based on our sample mutation queries, we can safely conclude that it should look like:
+
+```js
+type Mutation {
+  bookTrip(launchId: ID!): Boolean!
+  cancelTrip(launchId: ID!): Boolean!
+  login(email: String): String
+}
+```
+
+All the fields in the `Mutation` type returns a scalar type. The `bookTrip` field for booking a trip, `cancelTrip` field for cancelling a trip and the `login` field for authenticating a user.
+
+Now, put everything together in a `src/schema.js` file.
+
+_src/schema.js_
+
+```js
+const { gql } = require('apollo-server');
+
+const typeDefs = gql`
+  type Query {
+    launches(
+      """
+      the number of results to show. Must be >= 1
+      """
+      pageSize: Int
+      """
+      cursor - if you add a cursor id here, it will only return results _after_ this cursor
+      """
+      after: String
+    ): [Launch]!
+    launch(id: ID!): Launch
+    me: User
+  }
+  type Mutation {
+    bookTrip(launchId: ID!): Boolean!
+    cancelTrip(launchId: ID!): Boolean!
+    login(email: String): String
+  }
+  type Launch {
+    id: ID!
+    cursor: String
+    year: String!
+    date: String!
+    mission: Mission!
+    rocket: Rocket!
+    launchSuccess: Boolean
+    passengers: [User]!
+  }
+  type Rocket {
+    id: ID!
+    name: String!
+    type: String!
+  }
+  type User {
+    id: ID!
+    email: String!
+    trips: [Launch]!
+  }
+  type Mission {
+    name: String!
+    missionPatch: String
+  }
+`;
+
+module.exports = typeDefs;
+```
+
+In the code above, the `launches` field of the `Query` type takes in 2 arguments. Don't worry about those arguments for now, they are for pagination.
+
+<h2 id="apollo-server-run">Run your server</h2>
+
+Now that we have scoped out our app's schema, let's run the server. Copy the code below and paste in the `index.js` file.
+
+_src/index.js_
+
+```js
+...
+const typeDefs = require('./schema');
+
+const server = new ApolloServer({ typeDefs });
+
+server.listen().then(({ url }) => {
+  console.log(`🚀 Server ready at ${url}`);
+});
+
+```
+
+The schema `typeDefs` is passed to the `ApolloServer` constructor, then `ApolloServer`'s `listen()` method is invoked to run the server.
+
+On your terminal, run:
+
+```bash
+npm start
+```
+
+The `start` script in the `package.json` file should look like:
+
+```js
+...
+"scripts": {
+  "start": "nodemon src/index.js"
```

Comment:
@peggyrayzis I assume you'd also mention `nodemon` as part of the setup in the development section.